MobileCenterCrashes/MobileCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -181,23 +178,24 @@ - (void)testCreateBufferFile {
   // When
   NSString *testName = @"afilename";
   [self.sut createBufferFileWithName:testName forPriority:MSPriorityHigh];
-  
+
   // Then
-  NSString *priorityDirectory = [self.sut.logBufferDir stringByAppendingFormat:@"/%ld/", MSPriorityHigh];
 
+  // Wait for a little bit to make sure the files have been created.
+  [NSThread sleepForTimeInterval:0.1];
+  NSString *priorityDirectory = [self.sut.logBufferDir stringByAppendingFormat:@"/%d/", MSPriorityHigh];
   NSString *filePath =
       [priorityDirectory stringByAppendingPathComponent:[testName stringByAppendingString:@".mscrasheslogbuffer"]];
   BOOL success = [[NSFileManager defaultManager] fileExistsAtPath:filePath];
   XCTAssertTrue(success);
 }
 
-// FIXME: Crashes is getting way more logs than expected. Disable this functionality.
 - (void)emptyLogBufferFiles {
```

Comment:
Change the file name. `testEmptyLogBufferFiles`. I removed `test` prefix to prevent testing the method.